### PR TITLE
multi: fix errorlint warnings

### PR DESCRIPTION
**Pull request by dajohi**

### CODE REVIEW

File: client/rpcserver/handlers.go [hunk 1]
```diff
@@ -62,7 +62,7 @@ func createResponse(op string, res interface{}, resErr *msgjson.Error) *msgjson.
 // *msgjson.ResponsePayload.
 func usage(route string, err error) *msgjson.ResponsePayload {
 	usage, _ := commandUsage(route, false)
-	err = fmt.Errorf("%v\n\n%s", err, usage)
+	err = fmt.Errorf("%w\n\n%s", err, usage)
 	resErr := msgjson.NewError(msgjson.RPCArgumentsError, err.Error())
```
Comment (chappjc) [inline client/rpcserver/handlers.go:66]: This `err` shouldn't even be an `error` from what I can tell, it's just used for the string.

```suggestion
	resErr := msgjson.NewError(msgjson.RPCArgumentsError, "%v\n\n%s", err, usage)
```

File: client/rpcserver/types_test.go [hunk 2]
```diff
@@ -97,7 +97,7 @@ func TestParseNewWalletArgs(t *testing.T) {
 	for _, test := range tests {
 		nwf, err := parseNewWalletArgs(test.params)
 		if err != nil {
-			if !errors.Is(err, test.wantErr) {
+			if !errors.As(err, &test.wantErr) {
```
Comment (chappjc) [inline client/rpcserver/types_test.go:100]: This modifies `test.wantErr`.  That's pretty hard to swallow.  I think wrapping the other error value in types.go is making this awkward.  I don't think it's testing the right thing anymore either.
@JoeGruffins Can you provide feedback on the suggested changes to client/rpcserver/types.go with the other error wrapped?

**Review by chappjc — CHANGES_REQUESTED** [review]

File: client/rpcserver/types_test.go (hunk 2 above)
Comment (JoeGruffins) [reply client/rpcserver/types_test.go:100]: errArgs doesn't have a unique type, so yes this isn't testing what was intended anymore. Any standard error should unwrap into errArgs. I guess the linter doesn't want "%v" to be used with errors? The easy answer would be to just use the error string `err.Error()` for the second part of those errors and throw away the type. This is probably fine and won't change the current behavior. Another answer would be to give errArgs a type and an Unwrap method I think?

Comment (chappjc) [reply client/rpcserver/types_test.go:100]: Or just revert these types.go changes.

Comment (dajohi) [reply client/rpcserver/types_test.go]: reverted for now

**Review by chappjc — APPROVED** [review]
Looks good.  Just needs a test drive.

**Pull request merged**